### PR TITLE
Add GPIO ACPI and GPIO superIO support

### PR DESCRIPTION
**Pull request by piotr-kleins**
Signed-off-by: Piotr Kleinschmidt <piotr.kleins@gmail.com>

### CODE REVIEW

File: src/southbridge/amd/pi/hudson/acpi/fch.asl [hunk 1]
```diff
@@ -25,16 +25,17 @@
...
+	//CreateDWordField(Arg3,4,CDW2)
+	//CreateDWordField(Arg3,8,CDW3)
 
 	/* Check for proper PCI/PCIe UUID */
 	If(LEqual(Arg0,ToUUID("33DB4D5B-1FF7-401C-9657-7441C03DD766")))
 	{
 		/* Let OS control everything */
 		Return (Arg3)
 	} Else {
+		CreateDWordField(Arg3,0,CDW1)
```
Comment (miczyg1) [inline src/southbridge/amd/pi/hudson/acpi/fch.asl]: @piotr-kleins what are these changes for?

Comment (piotr-kleins) [reply src/southbridge/amd/pi/hudson/acpi/fch.asl]: ASL compiler gives warnings that CDW2 and CDW3 are unused, so I deleted them and made above part exactly the same way as it is in coreboot mainline

Comment (miczyg1) [reply src/southbridge/amd/pi/hudson/acpi/fch.asl]: @piotr-kleins the IASL compiler complained on it long time ago. Please revert back. We do not want weird behaviours on legacy.

Comment (piotr-kleins) [reply src/southbridge/amd/pi/hudson/acpi/fch.asl]: Done

**Pull request merged**